packages/web-client/tests/unit/models/workflow/workflow-session-test.ts

```diff
@@ -4,6 +4,8 @@ import WorkflowSession from '@cardstack/web-client/models/workflow/workflow-sess
 import WorkflowPersistence from '@cardstack/web-client/services/workflow-persistence';
 import Ember from 'ember';
 import BN from 'bn.js';
+import sinon from 'sinon';
+import Sinon from 'sinon';
```

Comment:
```suggestion
import { SinonFakeTimers } from 'sinon';
```

Maybe this would be a bit less strange-looking 🤔